sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 ## Release History
 
-## 4.0.0-beta.3 (Unreleased)
+## 4.0.0-beta.3 (2021-05-05)
```

Comment:
We cannot use `##` in our changelog, since this changelog content is used directly in the Microsoft Docs, these need to be set as H-2 level headers. This will break the Microsoft Docs page. 

So we need to use `###` for the version headers. And for the key bug fixes, new features, configuration renames, please use `####`. 

Please refer to changelog files for `azure-cosmos` and `azure-spring-data-cosmos` for more reference.

Reply:
Thanks for catching this @kushagraThapar